### ISSUE TITLE
schema-reader Error: The [web] middleware group has not been defined.

### ISSUE THREAD

**Issue by turkialawlqy** [VemtoOrg/vemto2-issues#80]
**Error Message:**
schema-reader Error: The [web] middleware group has not been defined.

**Stack:**
```
schema-reader Error: #0 /Users/turkialawlqy/Herd/Archive/vendor/laravel/jetstream/src/JetstreamServiceProvider.php(177): Illuminate\Foundation\Http\Kernel->appendMiddlewareToGroup('web', 'Laravel\\Jetstre...')
#1 /Users/turkialawlqy/Herd/Archive/vendor/laravel/jetstream/src/JetstreamServiceProvider.php(73): Laravel\Jetstream\JetstreamServiceProvider->bootInertia()
#2 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Laravel\Jetstream\JetstreamServiceProvider->boot()
#3 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#5 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure))
#6 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Container/Container.php(662): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL)
#7 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1099): Illuminate\Container\Container->call(Array)
#8 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1080): Illuminate\Foundation\Application->bootProvider(Object(Laravel\Jetstream\JetstreamServiceProvider))
#9 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(Laravel\Jetstream\JetstreamServiceProvider), 29)
#10 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1079): array_walk(Array, Object(Closure))
#11 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\Foundation\Application->boot()
#12 /Users/turkialawlqy/Herd/Archive/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(309): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#13 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/index.php(37): Illuminate\Foundation\Application->bootstrapWith(Array)
#14 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/common/Vemto.php(111): {closure}()
#15 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/index.php(18): Vemto\Vemto::execute('schema-reader', Object(Closure))
#16 /Applications/Vemto.app/Contents/Resources/static/VMTTL2(15): require('phar:///Applica...')
#17 {main}
```

---

Vemto 2.0.7
darwin 23.4.0
Locale: en-US
App version: 2.0.7

**Comment by TiagoSilvaPereira** [comment]
@turkialawlqy was this project created with Vemto or and existing one? Could you please check if the configuration is already cached?

```
php artisan config:cache
```

**Comment by TiagoSilvaPereira** [comment]
@turkialawlqy is this a new project or an existing one?

**Comment by TiagoSilvaPereira** [comment]
@turkialawlqy fixed on 2.0.10
